pkg/kubelet/kubelet.go

```diff
@@ -2065,6 +2064,8 @@ func (kl *Kubelet) HandlePodAdditions(pods []*v1.Pod) {
 		if !kl.podIsTerminated(pod) {
 			// Only go through the admission process if the pod is not
 			// terminated.
+		        
+                        existingPods := kl.podManager.GetPods()
```

Comment:
In this case, we are calling `GetPods` after we've called `podManager.AddPod`, whereas before that wasn't the case. Do you foresee that having an impact?

Reply:
thanks，you are right，I‘ll close it。